bin/mzml_statistics.py

```diff
@@ -77,13 +80,21 @@ def parse_bruker_d(file_name: str, file_columns: list):
             mslevel_map = {0: 1, 8: 2}
         elif 9 in df["MsMsType"].values:
             mslevel_map = {0: 1, 9: 2}
+        else:
+            msg = f"Unrecognized ms type '{df['MsMsType'].values}'"
+            raise ValueError(msg)
         df["MsMsType"] = df["MsMsType"].map(mslevel_map)
 
         try:
+            # This line raises an sqlite error if the table does not exist
+            _ = conn.execute("SELECT * from Precursors LIMIT 1").fetchall()
             precursor_df = pd.read_sql_query("SELECT * from Precursors", conn)
-        except Exception as e:
-            print(f"No precursers recorded in {file_name}")
-            precursor_df = pd.DataFrame()
+        except sqlite3.OperationalError as e:
+            if "no such table: Precursors" in str(e):
+                print(f"No precursers recorded in {file_name}, This is normal for DIA data.")
```

Comment:
**questions:** I guess this was originally written to write messages to stdout?

Reply:
well ... within this script the there is no real use of the stdout, so logging info goes into stdout and I assume that stderr will only contain tracebacks for raised errors ... 
I am not sure I 100% understand your question here ...